pep-0654.rst

```diff
@@ -1048,72 +1060,103 @@ clauses. We rejected this because it would be less obvious. The fact that we
 are handling ``ExceptionGroups`` of ``T`` rather than only naked ``Ts`` should be
 specified in the same place where we state ``T``.
 
-See Also
-========
 
-* An analysis of how exception groups will likely be used in asyncio
-  programs:
-  https://github.com/python/exceptiongroups/issues/3#issuecomment-716203284
+Programming Without except*
+===========================
 
-* The issue where the ``except*`` concept was first formalized:
-  https://github.com/python/exceptiongroups/issues/4
+Consider the following simple example of the ``except *`` syntax (suppose
+Trio natively supports this proposal):
```

Comment:
```suggestion
Consider the following simple example of the ``except *`` syntax (pretending
Trio natively supported this proposal):
```

(Also: it's a little unconventional to see the "efficient" version first and the "inefficient" version second. Usually we do before/after, where before is the old way and after is the new, improved way. :-)

Reply:
Yeah! I had the same thought, but then opted for my variant because of just how arcane the "inefficient" way looks like compared to `except*`. I'm afraid that readers would be lost in it and skip the whole section, not getting why we added it in the first place. :/